src/progress_bar_exercise/ProgressBar/progress-bar.component.js

```diff
@@ -0,0 +1,41 @@
+import { useCss } from "kremling";
+import React, { useEffect, useState } from "react";
+
+export default function ProgressBar({ loading }) {
+  const [seconds, setSeconds] = useState(0)
+  const [opacity, setOpacity] = useState(1)
```

Comment:
Seconds is kept in state to increment until 15.
Opacity will be changed to 0 for hiding of the progress bar after completion